src/test/java/org/folio/support/TestUtils.java

```diff
@@ -58,10 +62,45 @@ public static String readFile(String filePath) {
     return new String(Files.readAllBytes(getFile(filePath).toPath()));
   }
 
-  public record Link(UUID authorityId, String tag, String naturalId, char[] subfields) {
+  public static List<AuthorityDataStat> dataStatList(UUID userId1, UUID userId2) {
+    return List.of(
+      AuthorityDataStat.builder().id(UUID.randomUUID()).startedByUserId(userId1).build(),
+      AuthorityDataStat.builder().id(UUID.randomUUID()).startedByUserId(userId2).build()
```

Comment:
What about other data? I think it should be tested that all needed data is returned in the response.

Reply:
done!